src/index.html

```diff
@@ -7,12 +7,72 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card">
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="Apple iMac"
+      />
+
+      <div class="card__info">
+        <h1 class="card__title">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </h1>
+
+        <span class="card__label">Product code: 195434</span>
+
+        <div
+          class="card__review"
+          data-qa="card"
+        >
+          <ul class="stars stars--4">
+            <li class="stars__star"></li>
+
+            <li class="stars__star"></li>
+
+            <li class="stars__star"></li>
+
+            <li class="stars__star"></li>
+            
+            <li class="stars__star"></li>
```

Comment:
we can do it without spaces in this case. In my opinion - it will be more readable(Up to you)
```suggestion
            <li class="stars__star" />
            <li class="stars__star" />
            <li class="stars__star" />
            <li class="stars__star" />
            <li class="stars__star" />
```